codec/std/codec.go

```diff
@@ -1,7 +1,9 @@
 package std
```

Comment:
While doing this PR, can we move all this stuff to `std/`?

Reply:
Yes, but the std proto file itself might actually all go away if we really go through with this... The codec would remain l guess.